manage.py

```diff
@@ -254,10 +254,10 @@ def create_all():
     app = create_app(parse_options())
     log = logging.getLogger(__name__)
     with app.app_context():
-        tables_before = {t[0] for t in db.session.execute('SHOW TABLES')}
+        tables_before = db.engine.table_name()
```

Comment:
I don't have a strong preference. In my app I use:

``tables_after = [ t for t in db.engine.table_names() if t not in tables_before ]``

I will keep the return values as a set, i.e. ``tables_before = set(db.engine.table_names())`` so line 260 doesn't have to change.